puppet/modules/web/manifests/init.pp

```diff
@@ -62,6 +62,22 @@
     source => 'puppet:///modules/web/filter_apache_stats.sh',
   }
 
+  file { '/usr/local/bin/fastly-purge':
+    ensure => file,
+    owner  => 'root',
+    group  => 'root',
+    mode   => '0755',
+    source => 'puppet:///modules/web/fastly-purge.sh',
```

Comment:
It's preferred to inline files nowadays since it's faster in most cases. With source there's an additional HTTP request while with content it's part of the catalog. The performance difference shifts to the HTTP request above a certain size, but these scripts are small.
```suggestion
    content => file('web/fastly-purge.sh'),
```

Reply:
TIL